device-ecosystem/README.md

```diff
@@ -0,0 +1,57 @@
+# Device Ecosystem for DRA
+
+DRA is going to stable in 1.34 of Kubernetes. As more device vendors provide support for DRA, we want to have a tracking page where people can do to see who supports DRA.
+
+## Contributing
+
+We expect the status of the DRA drivers and their corresponds operators to change over time. Please feel free to add implementations as they graduate.
+
+This document is not official supported by Kubernetes and is mainly used by #wg-device-management to track where device vendors are in their integrations for DRA.
+
+## DRA Features
+
+One area that we want to track is not just DRA availability but also support of various DRA features as they graduate.
+
+### Table
+
+| Vendor  | DRA Driver
+|---------|------------
```

Comment:
The format can be vendor driver description with their landing page, list of supported features like a checklist of what is available from the upstream and supported hardware types. I can do an example entry for NVIDIA drivers